tests/config/test_config_sid.yml

```diff
@@ -30,7 +30,7 @@
       check_mode: yes
       register: sid_disabled
 
-    - name: Ensure netbios_name can't be changed without SID enabled.  # noqa 503
```

Comment:
I wonder why it is not an issue within the roles, as we use this idiom a lot there.

Reply:
That is the good question that I am not able to answer :-)
Maybe this is reported in tests only ...